Code/Framework/AzCore/AzCore/RTTI/BehaviorContext.cpp

```diff
@@ -321,6 +363,7 @@ namespace AZ
 
         if (m_method)
         {
+            m_method->PostProcessMethod(Base::m_context, *m_method);
```

Comment:
Let's call this ProcessMetaMethods, or ProcessAuxiliaryMethods, to indicate it checks for methods to always call before or after the main method when it is converted to a node.